docker-compose.yml

```
@@ -0,0 +1,21 @@
+version: "3.6"
+services:
+  blueoil:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+      # https://docs.nvidia.com/deeplearning/frameworks/user-guide/index.html#run_tf_cifar10
+      shm_size: "1gb"
+    image: blueoil_${USER}
+    ulimits:
+      memlock: -1
+      stack: 67108864
+    working_dir: /home/blueoil
+    volumes:
+      - ./:/home/blueoil
+      - ${DATA_DIR:-/storage/dataset}:/storage/dataset
+    environment:
+      - CUDA_VISIBLE_DEVICES=${CUDA_VISIBLE_DEVICES:-0}
+      - DATA_DIR=/storage/dataset
```

```suggestion
      - DATA_DIR=/home/blueoil/dataset
```
Same as above, `/home/blueoil/dataset` is better as default, I think.